rest-api/api/participant_counts_over_time_api.py

```diff
@@ -56,9 +56,9 @@ def validate_params(self, params):
       # Validate dates
       if not params['end_date']:
         raise BadRequest('end date should not be empty')
-      start_date = datetime.datetime.strptime('2017-01-01', DATE_FORMAT).date()
       try:
         end_date = datetime.datetime.strptime(params['end_date'], DATE_FORMAT).date()
+        start_date = end_date
```

Comment:
Does this cause problems if the start and end date are the same?

Reply:
The is for querying the data by a cutoff date instead of a date range. Technically, in this case the start_date should never be used, I assign a value for it just in case.